### PR TITLE
Two minimum viable agents

### PR DESCRIPTION
**Pull request by eob**
These are two agents which appear in the Agent Guidebook.

I was thinking it'd be good to make sure these are captured in the official codebase so that we could begin to write tests against them and then eventually find a way to import them **from this repo** into the website itself directly.

### CODE REVIEW

File: src/steamship/agents/examples/logs/shiplog--2023-07-15--00:02:11.log [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[Client] Creating/Fetching workspace with handle/id: default/None.
```
Comment (douglas-reid) [inline src/steamship/agents/examples/logs/shiplog--2023-07-15--00:02:11.log]: did you mean to include these logs?  should we add a .gitignore ?

File: src/steamship/agents/examples/chatgpt_api.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+class MyAssistant(AgentService):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self._agent = FunctionsBasedAgent(llm=ChatOpenAI(self.client), tools=[])
+
+
+if __name__ == "__main__":
+    AgentREPL(
+        MyAssistant,
+        agent_package_config={"botToken": "not-a-real-token-for-local-testing"},
```
Comment (douglas-reid) [inline src/steamship/agents/examples/chatgpt_api.py:21]: do you need `botToken` config here ?

File: src/steamship/agents/examples/logs/shiplog--2023-07-15--00:02:11.log (hunk 1 above)
Comment (eob) [reply src/steamship/agents/examples/logs/shiplog--2023-07-15--00:02:11.log]: Lol you were so fast with your slack notifications you caught the PR before I removed the logs :)

Added a .gitignore!

File: src/steamship/agents/examples/chatgpt_api.py (hunk 2 above)
Comment (eob) [reply src/steamship/agents/examples/chatgpt_api.py]: Argh. Sloppy of me. Thanks for the catch!

**Review by douglas-reid — APPROVED** [review]

**Pull request merged**